libs/structdiff/structtag/bundletag_test.go

```diff
@@ -0,0 +1,43 @@
+package structtag
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBundleTagMethods(t *testing.T) {
+	tests := []struct {
+		tag          string
+		isReadOnly   bool
+		isInternal   bool
+		isDeprecated bool
+	}{
+		// only one annotation.
+		{"readonly", true, false, false},
```

Comment:
This would be more readable as 

`
{
    tag: "readonly"
    isReadOnly: true,
    // The false ones can and should be be omitted.
}
`